Description: Fixes get_form() uses kwargs
 In Django 1.10, get_from() needs to use kwargs instead of a normal param.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-07-20

--- horizon-10.0.0~b2.orig/horizon/forms/views.py
+++ horizon-10.0.0~b2/horizon/forms/views.py
@@ -165,8 +165,10 @@ class ModalFormView(ModalFormMixin, view
         """
         return obj.name
 
-    def get_form(self, form_class):
+    def get_form(self, form_class=None):
         """Returns an instance of the form to be used in this view."""
+        if form_class is None:
+            form_class = self.get_form_class()
         return form_class(self.request, **self.get_form_kwargs())
 
     def form_invalid(self, form):
